### ISSUE TITLE
1d signal feature learning

### ISSUE THREAD

**Issue by LudoAcciai** [vlfeat/matconvnet#516]
Is it possible to use matconvnet for 1d signal unsupervised feature learning?

**Comment by lenck** [comment]
yeah, sure, should be possible. You may need to implement your loss, but for example and autoencoder should be pretty straight-forward to implement...
